lmod_proxy/cmd.py

```diff
@@ -1,22 +1,17 @@
 # -*- coding: utf-8 -*-
 """Command line debug start of flask application"""
-import logging
 import os
 
-from lmod_proxy import web
-
-log = logging.getLogger('lmod_proxy')
+from lmod_proxy import config
 
 
 def run_server():
-    logging.basicConfig(level=logging.DEBUG)
+
```

Comment:
sure, it doesn't have any tests either, since this is just for debug running, but I'll at least add a doc string